catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -301,12 +288,34 @@ public void testExternalFileset() throws IOException {
 
   @Test
   void testNameSpec() {
-    String illegalName = "/%~?*";
+    String illegalName = "ok/test";
+
+    // test illegal catalog name
+    Exception exception =
+        Assertions.assertThrows(
+            IllegalArgumentException.class,
+            () -> metalake.createCatalog(illegalName, Catalog.Type.FILESET, provider, null, null));
+    Assertions.assertTrue(exception.getMessage().contains("The catalog name 'ok/test' is illegal"));
+
+    // test illegal schema name
+    exception =
+        Assertions.assertThrows(
+            IllegalArgumentException.class,
+            () -> catalog.asSchemas().createSchema(illegalName, "comment", null));
+    Assertions.assertTrue(
+        exception.getMessage().contains("does not support '/' in the name for SCHEMA"));
 
+    // test illegal fileset name
     NameIdentifier nameIdentifier = NameIdentifier.of(schemaName, illegalName);
-
-    Assertions.assertThrows(
-        NoSuchFilesetException.class, () -> catalog.asFilesetCatalog().loadFileset(nameIdentifier));
+    exception =
+        Assertions.assertThrows(
+            IllegalArgumentException.class,
+            () ->
+                catalog
+                    .asFilesetCatalog()
+                    .createFileset(nameIdentifier, null, MANAGED, null, null));
+    Assertions.assertTrue(
+        exception.getMessage().contains("does not support '/' in the name for FILESET"));
```

Comment:
Can you please also add a test to verify the fileset rename to a illegal name?

Reply:
added